src/multiPortConnector.js

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright 2022, Tony Atkins
+ * Copyright 2011-2020, Colin Clark
+ *
+ *  Licensed under the MIT license, see LICENSE for details.
+ */
+
+(function (fluid) {
+    "use strict";
+    var youme = fluid.registerNamespace("youme");
+
+    // Enclosing grade so that we can recreate our pool of connections.
+    fluid.defaults("youme.multiPortConnector.connections", {
+        gradeNames: ["fluid.component"],
+
+        invokers: {
+            open: {
+                funcName: "youme.multiPortConnector.callAllChildInvokers",
+                args: ["{that}", "youme.connection", "open"] // that, gradeName, invokerName, invokerArgs
+            },
+
+            close: {
+                funcName: "youme.multiPortConnector.callAllChildInvokers",
+                args: ["{that}", "youme.connection", "close"] // that, gradeName, invokerName, invokerArgs
+            }
+        }
+    });
+
+    fluid.defaults("youme.multiPortConnector.connections.inputs", {
+        gradeNames: ["youme.multiPortConnector.connections"],
+
+        dynamicComponents: {
+            connection : {
+                type: "youme.connection.input",
+                sources: "{connections}.options.sources",
+                options: {
+                    openImmediately: true,
+                    portSpec: "{source}.portSpec",
+                    members: {
+                        port: "{source}.port"
+                    }
+                }
+            }
+        }
+    });
+
+    fluid.defaults("youme.multiPortConnector.connections.outputs", {
+        gradeNames: ["youme.multiPortConnector.connections"],
+
+        dynamicComponents: {
+            connection : {
```

Comment:
Odd stray space here, funny the linter doesn't catch it

Reply:
There's a rule, but it's not enabled by default:

https://eslint.org/docs/rules/key-spacing

I've enabled it in my `.eslintrc.json` for now.  We can make a ticket for eslint-config-fluid if this is something we do consistently in a range of projects.

Reply:
The defaults also enforce only a single space after the colon, which is not always my preferred style, but if I can give up ellipses in all my test messages, I can also try that.